packages/react-core/src/components/Form/FormGroup.tsx

```diff
@@ -102,8 +105,15 @@ export const FormGroup: React.FunctionComponent<FormGroupProps> = ({
       )}
 
       <div className={css(styles.formGroupControl, isInline && styles.modifiers.inline)}>
+        {helperTextIsBeforeField &&
+          (validated === ValidatedOptions.error && helperTextInvalid
+            ? inValidHelperText
+            : showValidHelperTxt(validated))}
```

Comment:
Add
```ts
const helperText = validated === ValidatedOptions.error && helperTextInvalid
            ? inValidHelperText
            : showValidHelperTxt(validated);
```

Then change to `{helperTextIsBeforeField && helperText}` and `{!helperTextIsBeforeField && helperText}`.